docs/reference/kubectl/overview.md

```diff
@@ -192,6 +192,29 @@ NAME           RSRC
 submit-queue   610995
 ```
 
+#### Server-side columns
+
+`kubectl` supports receiving specific column information from the server about objects.
```

Comment:
Why `kubectl` users need to care about this?

Reply:
In the near future this will be the default and only way of interacting with the server. We're putting it here to gather feedback about the feature, especially the inconsistencies between the current behavior of kubectl.